rotkehlchen/db/upgrades/v46_v47.py

```diff
@@ -52,7 +80,24 @@ def _remove_nft_collection_assets(write_cursor: 'DBCursor') -> None:
         placeholders = ','.join(['?'] * len(identifiers_to_remove))
         with globaldb_conn.write_ctx() as global_db_write_cursor:
             log.debug('Deleting from globaldb...')
-            global_db_write_cursor.execute(f'DELETE FROM assets WHERE identifier IN ({placeholders})', identifiers_to_remove)  # noqa: E501
+            global_db_write_cursor.executescript('PRAGMA foreign_keys = OFF;')
```

Comment:
Explain why you are doing it like this. Say that cascading deletion with Foreign keys ON took `X` mins for your DB. While this approach, essentially manually managing the cascading deletion took `X` mins for your DB.